app/components/market/submit/submit-market-application.component.ts

```diff
@@ -61,6 +61,19 @@ export class SubmitMarketApplicationComponent implements OnInit {
         this.modeState = mode;
     }
 
+    public get submitToolTip(): string {
+        if (this.isFormValid()) {
+            return "Click to submit form";
```

Comment:
Change to "Click to submit job"

Reply:
This tooltip text will also be displayed when user is in pool only mode.